shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/MetaData.hs

```diff
@@ -32,13 +31,12 @@ import           Shelley.Spec.Ledger.Serialization (mapFromCBOR, mapToCBOR)
 
 -- | A generic metadatum type.
 --
--- TODO make strict
 data MetaDatum
-    = Map [(MetaDatum, MetaDatum)]
-    | List [MetaDatum]
-    | I Integer
-    | B B.ByteString
-    | S T.Text
+    = Map ![(MetaDatum, MetaDatum)]
+    | List ![MetaDatum]
```

Comment:
FYI, this data type is on my list to make strict (I added the TODO). The use of `[]` and `(,)` prevent it from being truly strict. I plan to address it later when we have strict variants of those types (WIP).

Reply:
@mrBliss I can drop that patch from this PR if you like.

Reply:
@mrBliss I droppped the TODO fix from this PR.